profiles/models.py

```diff
@@ -7,6 +7,24 @@
 from jsonfield import JSONField
 
 
+class Employment(models.Model):
+    """
+    A user work_history
+    """
+    city = models.TextField()
+    company_name = models.TextField()
+    country = models.TextField()
+    industry = models.TextField()
+    position = models.TextField()
+    state_or_territory = models.TextField(blank=True, null=True)
+    end_date = models.DateTimeField()
```

Comment:
Also, can you add back `blank=True, null=True`? I realized that this is needed to support the case where the user is currently working and there is no end date